surfactant/cmd/generate.py

```diff
@@ -223,6 +225,13 @@ def get_default_from_config(option: str, fallback: Optional[Any] = None) -> Any:
     is_eager=True,
     help="List supported input formats",
 )
+@click.option(
+    "--include_all_files",
+    is_flag=True,
+    default=False,
```

Comment:
```suggestion
    default=get_default_from_config("include_all_files"),
```

Let's pull the default value from the `core.include_all_files` config option, so users can change the default behavior. If they haven't set anything, I think the default fallback value returned is `NoneType`, which should be considered false-y for comparisons... or `fallback=False` could be added as an additional argument to `get_default_from_config`.